### PR TITLE
feat(read) : #MAG-504 add board reading view

### PR DESCRIPTION
**Pull request by FlorentMr**
## Describe your changes
add board reading view

## Issue ticket number and link
[MAG-504](https://jira.support-ent.fr/browse/MAG-504)

## Checklist before requesting a review (magic string, indentation, comment/documentation...)

- [x] I have detailed the tests to do in my feature/fix in order to prevent consequents regressions (must specify in **Checklist tests**)
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (API Doc etc...) - (must specify in **Description** for target version)
- [ ] If it is a consequent feature, I have added thorough tests.
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Any dependent changes have been added to this project (must specify in **Description**)

### CODE REVIEW

File: frontend/src/components/board-explorer/BoardExplorer.tsx [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+        type="submit"
+        variant="filled"
+        className="retour-button"
+        onClick={navigateToView}
+        style={retourStyle}
+      >
+        <IconButton sx={iconButtonReturnStyle}>
+          <ArrowBackIcon sx={{ color: "white" }} />
+        </IconButton>
+        {"Retour"}
```
Comment (niko-amosse) [inline frontend/src/components/board-explorer/BoardExplorer.tsx:200]: i18n

**Review by niko-amosse — APPROVED** [review]

File: frontend/src/components/board-explorer/BoardExplorer.tsx [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+
+  useEffect(() => {
+    setSection(filteredSections[0]);
+  }, [filteredSections]);
+
+  const navigatePage = (direction: "next" | "prev") => {
+    const newIndex = direction === "next" ? cardIndex + 1 : cardIndex - 1;
+    const newCard = initialCards[newIndex];
+
+    if (board.isLayoutFree()) {
```
Comment (alicedraillard) [inline frontend/src/components/board-explorer/BoardExplorer.tsx:100]: Pr creer une section pr le freelayout ?

File: frontend/src/components/board-explorer/BoardExplorer.tsx [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+      <Button
+        color="primary"
+        type="submit"
+        variant="filled"
+        className="retour-button"
+        onClick={navigateToView}
+        style={retourStyle}
+      >
+        <IconButton sx={iconButtonReturnStyle}>
+          <ArrowBackIcon sx={{ color: "white" }} />
```
Comment (alicedraillard) [inline frontend/src/components/board-explorer/BoardExplorer.tsx:198]: rajouter style

File: frontend/src/components/board-explorer/BoardExplorer.tsx [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+      </Button>
+
+      <Box sx={contentStyle}>
+        <Box sx={boxStyle}>
+          {cardIndex > 0 && (
+            <IconButton
+              sx={leftNavigationStyle}
+              onClick={() => navigatePage("prev")}
+            >
+              <ChevronLeftIcon sx={{ color: "black" }} />
```
Comment (alicedraillard) [inline frontend/src/components/board-explorer/BoardExplorer.tsx:210]: idem style

File: frontend/src/components/board-explorer/BoardExplorer.tsx [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+            </IconButton>
+          )}
+          {!isLastCardInBoard() && (
+            <IconButton
+              sx={rightNavigationStyle}
+              onClick={() => navigatePage("next")}
+            >
+              <ChevronRightIcon sx={{ color: "black" }} />
```
Comment (alicedraillard) [inline frontend/src/components/board-explorer/BoardExplorer.tsx:218]: idem

**Pull request merged**